components/organisms/ContributorsTable/contributors-table.tsx

```diff
@@ -23,11 +23,11 @@ const ContributorTable = ({
 
       {contributors &&
         contributors.length > 0 &&
-        contributors.map((contributor, i) => (
+        contributors.map((contributor) => (
           <ContributorListTableRow
             topic={topic}
             contributor={contributor}
-            key={i}
+            key={contributor.user_id}
```

Comment:
This is a guaranteed unique reliable key for the row. If the items were to be reordered, the original index, `i`, would not remain with the same key.